todo/src/reducers/todos.js

```diff
@@ -0,0 +1,50 @@
+import { ADDTODO, REMOVE_TODO, TOOGLE_COMPLETE, IMPORT_STATE } from '../actions/todoActions';
+
+export default (todos = [], action) => {
+	switch (action.type) {
+    case ADDTODO:
+      todos = [...todos, action.payload];
+      window.localStorage.setItem('todos', JSON.stringify(todos));  
+      return todos;
+    case REMOVE_TODO:
+      const id = Number(action.payload);
+      todos = [...todos.slice(0, id), ...todos.slice(id + 1)];
+      window.localStorage.setItem('todos', JSON.stringify(todos));
```

Comment:
instead of setting the localStorage every time the todo list is updated it may be more efficient to update local storage only when the component unmounts.